stix_shifter_modules/nozomi/stix_translation/query_constructor.py

```diff
@@ -0,0 +1,549 @@
+                                                 ComparisonComparators.IsSubSet):
+            raise NotImplementedError('Nozomi is not supported for NOT <, NOT >, NOT <=, NOT >=, NOT ISSUBSET'
+                                      ' operators')
+
+        if mapped_field_type == "bytes" and comparator in (ComparisonComparators.Equal, ComparisonComparators.NotEqual,
+                                                           ComparisonComparators.In):
+            value = f"{value} bytes"
+
+        return value
+
+    def _lookup_comparison_operator(self, expression_operator) -> str:
+        """
+        lookup operators support in nozomi
+        :param expression_operator:object
+        :return str
+        """
+        if str(expression_operator) not in self.comparator_lookup:
+            raise NotImplementedError(
+                f'Comparison operator {expression_operator.name} unsupported for nozomi connector')
+
+        return self.comparator_lookup[str(expression_operator)]
+
+    def _eval_comparison_value(self, expression, mapped_field_type, mapped_fields_array) -> str:
+        """
+        Function for parsing comparison expression value
+        :param expression: expression object
+        :param mapped_field_type: str
+        :param mapped_fields_array: list object
+        :return: formatted expression value
+        """
+        if expression.comparator == ComparisonComparators.Like:
+            value = self._check_value_comparator_support(expression.value, expression.comparator, mapped_field_type,
+                                                         mapped_fields_array, expression)
+            value = self._format_value(value)
+        elif expression.comparator == ComparisonComparators.In:
+            value = self._format_set(expression.value, mapped_field_type, expression, mapped_fields_array)
+        elif expression.comparator in [ComparisonComparators.GreaterThan, ComparisonComparators.GreaterThanOrEqual,
+                                       ComparisonComparators.LessThan, ComparisonComparators.LessThanOrEqual,
+                                       ComparisonComparators.Equal, ComparisonComparators.NotEqual]:
+            value = self._check_value_comparator_support(expression.value, expression.comparator, mapped_field_type,
+                                                         mapped_fields_array, expression)
+            value = self._format_value(value)
+        elif expression.comparator == ComparisonComparators.IsSubSet:
+            value = self._check_value_comparator_support(expression.value, expression.comparator, mapped_field_type,
+                                                         mapped_fields_array, expression)
+            value = self._format_value(value)
+        else:
+            raise NotImplementedError('Unknown comparator expression operator')
+        return value
+
+    def _add_qualifier(self, query, qualifier) -> list:
```

Comment:
Is the only qualifer that is supported time range?

Reply:
record_created_at is the only qualifier supported by Data Source. https://technicaldocs.nozominetworks.com/products/n2os/sdk/open-api-query.html